pkg/sloop/ingress/kubewatcher.go

```diff
@@ -306,14 +306,22 @@ func (i *kubeWatcherImpl) processUpdate(kind string, obj interface{}, watchResul
 	glog.V(99).Infof("processUpdate: obj json: %v", resourceJson)
 
 	kubeMetadata, err := kubeextractor.ExtractMetadata(resourceJson)
+	EventInfo, err1 := kubeextractor.ExtractEventInfo(resourceJson)
+	InvolvedObject, err2 := kubeextractor.ExtractInvolvedObject(resourceJson)
 	if err != nil || kubeMetadata.Namespace == "" {
 		// We are only grabbing namespace here for a prometheus metric, so if metadata extract fails we just log and continue
 		glog.V(2).Infof("No namespace for resource: %v", err)
 	}
-	metricIngressKubewatchcount.WithLabelValues(kind, watchResult.WatchType.String(), kubeMetadata.Namespace).Inc()
-	metricIngressKubewatchbytes.WithLabelValues(kind, watchResult.WatchType.String(), kubeMetadata.Namespace).Add(float64(len(resourceJson)))
```

Comment:
Since we will be scraping metrics endpoint into prometheus of HLO, we should be able to view all the metrics that get scraped into it persisting to the retention period. Is that what you were referring to?

Reply:
Yes as far as I know we can get aggregate metrics through PromQL , find the attached screenshots.
<img width="1786" alt="Screenshot 2022-07-12 at 11 16 16 PM" src="https://user-images.githubusercontent.com/109100113/178559023-d13299c4-68c9-4a92-bce8-5ba8e1852c7d.png">

<img width="1775" alt="Screenshot 2022-07-12 at 11 15 53 PM" src="https://user-images.githubusercontent.com/109100113/178558996-8e02cb4c-68d5-46eb-b8da-c1c3901a00d1.png">